about/policy/open-sgid/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Open SGID Policy and Terms of Use
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+tags:
+  - policy
+  - sgid
+categories: []
+date: 2020-05-04 10:01:27 -0500
+---
+
+The following guidelines apply when using the Open SGID database. 
+
+## Do
+{: .text-left }
+
+- Search our [known-issues](https://github.com/agrc/open-sgid/issues) page if you are having trouble 
+- {% include contact.html subject=page.title text="Contact " contact=site.data.contacts.opensgid %} if you encounter an new issue
+- Use for individual purposes only. Examples include:
+  - Cartography
+  - Geoprocessing
+  - Analysis
+  - Scripting
+
+## Don't
+{: .text-left }
+
+- Create services from layers
+- Develop large-scale applications using Open SGID
```

Comment:
**suggestion**: i would clarify this. I think it is quite nebulous. Does it mean don't use open sgid data in the creation of a large-scale application at all? What is the application scale? Are we mainly saying not not use a live connection to the open sgid live applications?

Reply:
yeah, i'm struggling with this one, too.  from my understanding, we don't want users to build applications (apps) that heavily hit the database with calls.  is that what the idea with this one?

Reply:
create processes that make excessive database calls  (?)

Reply:
i reworded it to say "create processes that make excessive database calls".  but, for now, I'll leave it open for discussion until i hear some chatter.

Reply:
i don't know remember. i think that was a @brigsz. Maybe it's covered by the   map and web service line?

Reply:
i'm going to mark this as resolved, as it's outdated.  we can open this again if needed using the updated wording... "create processes that make excessive database calls"